webapp-django/crashstats/crashstats/models.py

```diff
@@ -141,6 +141,37 @@ class Meta:
         ordering = ['sort']
 
 
+class ProductVersion(models.Model):
+    """Lookup table for product versions and build information"""
+    product_name = models.CharField(
+        max_length=50, blank=False, null=False,
+        help_text='ProductName of product as it appears in crash reports'
+    )
```

Comment:
This is not a foreign key because I want this table to stand alone and be easy to insert things into and look things up without dealing with joins. We're using it solely for the `BetaVersionRule`. Once Buildhub2 is up, we could ditch this and use Buildhub2 exclusively.